Tests/LibWeb/Layout/input/block-and-inline/float-max-content-containing-block.html

```diff
@@ -0,0 +1,14 @@
+<style>
+    div {
+        float: left;
+    }
+
+    span {
+        float: left;
+    }
+</style>
+
+<div>
+    <span>A</span>
+    <span>B</span>
+</div>
```

Comment:
let's try to make layout dumps as small as we can :)
```suggestion
</style><div><span>A</span><span>B</span></div>```

Reply:
Done.